src/components/AddExpensesForm.js

```diff
@@ -0,0 +1,91 @@
+import { useState } from 'react';
+import { v4 as uuid } from 'uuid';
+
+const AddExpensesForm = () => {
+  const initialState = {
+    id: '',
+    name: '',
+    date: '',
+    amount: '',
+  };
+
+  const [singleExpense, setSingleExpense] = useState(initialState);
+  const [expenses, setExpenses] = useState([]);
+
+  const handleChange = (evt) => {
+    const { name, value } = evt.target;
+    setSingleExpense({
+      ...singleExpense,
+      [name]: value,
+    });
+  };
+
+  const handleSubmit = (evt) => {
+    evt.preventDefault();
+    const newExpenses = [...expenses, { ...singleExpense, id: uuid() }];
+    setExpenses(newExpenses);
+    setSingleExpense(initialState);
+  };
+
+  return (
+    <div className="container-md">
```

Comment:
do we want to add the container to each component or rather to the whole page?

Reply:
Yes indeed! I'll change it to `row`. The best option for me personally would be to create a `<Layout />` component which we could use as a main container and pass all other necessary components as children...